### ISSUE TITLE
Server stops

### ISSUE THREAD

**Issue by daniktheboss** [HotFireyDeath/Glifcos#6]
Glifcos loading
Glifcos enabling 
Server stopped?? 
What's the problem?

**Comment by Patra3** [comment]
Is there a crash log? Does it say anything when it crashes? Please provide some more information, as there are no crashes for me...

**Comment by daniktheboss** [comment]
[12:00:31] [Server thread/INFO]: 

```
       PocketMine-iTX Genisys is a fork of PocketMine-MP, made by iTX Technologies LLC.
       Version: 1.2dev-596
       Target client Version: v0.15.0 alpha
       Latest source code is available at https://github.com/iTXTech/Genisys
       Donate link: http://pl.zxda.net/plugins/203.html
```

[12:00:31] [Server thread/INFO]: Loading pocketmine.yml...
[12:00:31] [Server thread/INFO]: Loading genisys.yml...
[12:00:32] [Server thread/INFO]: Loading server properties...
[12:00:32] [Server thread/INFO]: Selected English (eng) as the base language
[12:00:32] [Server thread/INFO]: Starting Minecraft: PE server version v0.15.0 alpha
[12:00:32] [Server thread/INFO]: Opening server on 0.0.0.0:19132
[12:00:32] [Server thread/INFO]: This server is running Genisys version 1.2dev-596 "Ikaros" (API 2.0.0)
[12:00:32] [Server thread/INFO]: Genisys is distributed under the GPL License version 3 and later
[12:00:33] [Server thread/ERROR]: Could not load plugin 'Glifcos-p': plugin exists
[12:00:33] [Server thread/INFO]: Loading Glifcos-p v0.1.2
[12:00:33] [Server thread/INFO]: Preparing level "world"
[12:00:33] [Server thread/INFO]: Preparing level "nether"
[12:00:33] [Server thread/INFO]: Enabling Glifcos-p v0.1.2
[PE Server] Server was stopped.

**Comment by Patra3** [comment]
Huh. Ok.
I'd recommend you stop using Glifcos (remove the plugin) for now, I will investigate this issue and give you a reply later.

**Comment by daniktheboss** [comment]
Ok thanks! BTW this is on android... I'll check on my main server and let u know

**Comment by daniktheboss** [comment]
Ok maybe I'm doing somthing wrong .. you are supposed to make the zip as phar?

**Comment by Patra3** [comment]
http://glifcos-cs.weebly.com/d-setup-guide.html

**Comment by daniktheboss** [comment]
Wait... ur suppost to have a Web server? Like a website?

**Comment by daniktheboss** [comment]
Ok I got it! I have a localhost server but don't get how to put ur localhost ip

**Comment by Patra3** [comment]
You can do something like this in your config.yml file for localhost:
http://localhost/directory/something/etc

**Comment by daniktheboss** [comment]
Ok it now says failed to open stream

**Comment by Patra3** [comment]
All issues are now fixed in the 2.0 release.
https://github.com/HotFireyDeath/Glifcos/releases/tag/2.0.0-b1
